f8a_worker/workers/dependency_parser.py

```diff
@@ -144,3 +148,17 @@ def get_npm_dependencies(cls, path):
                                       package=name, version=version))
 
         return set_package_names
+
+    @staticmethod
+    def change_package_lock_to_shrinkwrap():
+        """Rename package-lock.json to npm-shrinkwrap.json.
+
+        For more information about package-lock.json please visit
+        https://docs.npmjs.com/files/package-lock.json
+        """
+        # TODO: Remove this method once mercator has support for package-lock.json
+
+        package_lock_path = Path.cwd() / "package-lock.json"
```

Comment:
Does this reliably work? I mean, can we expect CWD to be correct when we hit this line?

Reply:
Yes, because we are in correct directory in line 43 itself.

Reply:
I see it now, thanks :wink: